rpi_backlight.py

```diff
@@ -100,7 +104,20 @@ def set_power(on: bool) -> None:
     """
 
     # 0 is on, 1 is off
-    _set_value("bl_power", int(not on))
+    if int(_get_value("tinker_mcu_bl")) == 0:
+        if on == True:
+            _set_value("tinker_mcu_bl", 255)
+    else:
+        if on == False:
+            _set_value("tinker_mcu_bl", 0)
+
+def toggle_power():
+    """Toggle the display power on or off.
+    """
+    if int(_get_value("tinker_mcu_bl")) == 0:
```

Comment:
I need clarification about the display. Does it only work with brightness, not real power off?

Reply:
Here is the output. Please let me know next steps.

total 0
drwxr-xr-x 3 root root 0 Apr 6 20:56 .
drwxr-xr-x 6 root root 0 Feb 13 20:42 ..
-r--r--r-- 1 root root 4096 Apr 6 20:32 modalias
-r--r--r-- 1 root root 4096 Apr 6 20:32 name
lrwxrwxrwx 1 root root 0 Apr 6 20:32 of_node -> ../../../../../firmware/devicetree/base/i2c@ff150000/tinker_mcu@45
drwxr-xr-x 2 root root 0 Apr 6 20:32 power
lrwxrwxrwx 1 root root 0 Feb 13 20:42 subsystem -> ../../../../../bus/i2c
-rw-r--r-- 1 root root 4096 Feb 13 20:42 uevent
root@TinkerBoardS2:/sys/bus/i2c/drivers/tinker_mcu #